build/util/util.sh

```diff
@@ -61,7 +61,7 @@ push_aws () {
     echo "pushing argo-cd backup to aws"
     BACKUP_BUCKET_NAME=`cat /secrets/aws.bucket.name`
     BACKUP_BUCKET_URI="s3://${BACKUP_BUCKET_NAME}"
-    aws s3 mb ${BACKUP_BUCKET_URI}
+    aws s3 mb ${BACKUP_BUCKET_URI} --region us-east-1
```

Comment:
Fixes #279

Reply:
It still attempts to create a bucket though. I can't imagine any single enterprise environment where this isn't going to be a major red flag.

Also, it's an implicit behavior not necessarily easy to find, so I'd strongly recommend to get rid of it completely.

Reply:
Hardcoding `--region us-east-1` is probably a bad idea.

Reply:
If you send your create bucket request to the s3.amazonaws.com endpoint, the request go to the us-east-1 region by default.  

The behavior of Argo CD Export has always been to create bucket in us-east-1, This change is required as the new version of AWS CLI requires a region to be provided. This can be parameterized to allow users to provide region of their choice but I don't see much use.

Reply:
Default is one thing. There should be a default but hardcoding to always use us-east-1 is the right thing to do?

Reply:
@wtam2018 
sorry I missed this one, 

so previously, the command to create s3 bucket using the aws CLI was 
`aws s3 mb <foo>` -> which would create the bucket `foo` in `us-east-1` region.

But in the new version of AWS CLI, the command got changed to 
`aws s3 mb <foo> --region <us-east-1>` 

I tried to keep the behavior similar.

Are you saying we should parameterize the region using some command line arguments ? and may be keep the default(if nothing is provided by the user) as `us-east-1` ?

Reply:
Yes. I am suggesting to parameterize the region.

Reply:
@wtam2018 made requested changes. PTAL